metrics/density/docker_memory_usage.sh

```diff
@@ -251,6 +252,10 @@ EOF
 			die "Failed to find PSS for $HYPERVISOR_PATH"
 		fi
 
+		virtiofsd_mem="$(get_pss_memory "$VIRTIOFSD_PATH")"
+		if [ "$virtiofsd_mem" == "0" ]; then
+			die "Failed to find PSS for $VIRTIOFSD_PATH"
```

Comment:
I'm not sure in the case of `virtiofsd` that this should be a `die` condition. I don't think every user will have virtiofsd installed or in use will they? Maybe just make this a `warn`?.  The others like proxy and shim are `die` conditions, as they are expected to be present (although, with the advent of shimv2 etc. these rules are probably not quite as strong and useful as they used to be).

Reply:
ok, that' better, I will change "die" to "echo".

Reply:
I'd probably use `warning`. We have `die` and `warning` [in the test lib code](https://github.com/kata-containers/tests/blob/master/metrics/lib/common.bash#L31-L45). Elsewhere in kata we also have `info` (but maybe not in the tests code).